app/src/main/java/edu/uwstout/p2pchat/SettingActionListeners/HelpPageListener.java

```diff
@@ -0,0 +1,22 @@
+package edu.uwstout.p2pchat.SettingActionListeners;
+
+import android.view.View;
+
+/**
+ * Class responsible for Help item an actiion.
+ */
+public class HelpPageListener implements View.OnClickListener
+{
+
+    /**
+     * A transition to the help page fragment.
+     *
+     * @param view
+     *         view that was just clicked.
+     */
+    @Override
+    public void onClick(View view)
+    {
+        //TODO ADD NAVIGATION TO HELP PAGE HERE.
```

Comment:
I would prefer a Toast to say that implementation is pending, but this is fine for now.